### PR TITLE
Update libs and tests to work with latest version of truffle and web3

### PR DESCRIPTION
**Pull request by adklempner**

### CODE REVIEW

File: libs/utils/misc.js [hunk 1]
```diff
@@ -102,7 +102,7 @@ function lifWei2Lif(value, context){
 };
 
 function lif2LifWei(value, context){
-  return context.web3.utils.toWei(value, 'ether');
+  return context.web3.utils.toWei(''+value, 'ether');
```
Comment (AugustoL) [inline libs/utils/misc.js]: why add the ''?

File: test/HotelManager.js [hunk 2]
```diff
@@ -31,8 +31,8 @@ describe('HotelManager', function() {
     ownerAccount = wallet["0"].address;
     daoAccount = wallet["1"].address;
 
-    await utils.fundAccount(fundingSource, ownerAccount, 50, web3);
-    await utils.fundAccount(fundingSource, daoAccount, 50, web3);
+    await utils.fundAccount(fundingSource, ownerAccount, '50', web3);
```
Comment (AugustoL) [inline test/HotelManager.js]: cant we parse the integer to string if needed in `fundAccount` ?

File: test/helpers/token.js [hunk 3]
```diff
@@ -90,7 +90,7 @@ async function simulateCrowdsale(rate, balances, accounts, weiPerUSD) {
     if (balances[i] > 0){
       const options = {
         to: crowdsale.options.address,
-        value: web3.utils.toWei(balances[i]/rate, 'ether'),
+        value: web3.utils.toWei(web3.utils.toBN(i+1), 'ether'),
```
Comment (AugustoL) [inline test/helpers/token.js]: here we shouldn't divide for `rate`?

File: libs/utils/misc.js (hunk 1 above)
Comment (adklempner) [reply libs/utils/misc.js]: New version of web3js enforces using String/BigNumber for precision.

File: test/HotelManager.js (hunk 2 above)
Comment (adklempner) [reply test/HotelManager.js]: We can. It looks like we only use `fundAccount` for tests, so it wouldn't make a big difference either way. If you think it's better practice I can change it.

File: test/helpers/token.js (hunk 3 above)
Comment (AugustoL) [reply test/helpers/token.js]: :+1:

File: test/HotelManager.js (hunk 2 above)
Comment (AugustoL) [reply test/HotelManager.js]: Ok, so lets leave it that way!

File: libs/utils/misc.js (hunk 1 above)
Comment (AugustoL) [reply libs/utils/misc.js]: :+1:

**Comment by AugustoL** [comment]
:clap:

**Pull request merged**